docs/extension-system/viewer-editor-apps.md

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Viewer and editor apps'
+date: 2024-03-12T00:00:00+00:00
+weight: 60
+geekdocRepo: https://github.com/owncloud/web
+geekdocEditPath: edit/master/docs/extension-system
+geekdocFilePath: viewer-editor-apps.md
+geekdocCollapseSection: true
+---
+
+{{< toc >}}
+
+## Viewer and editor apps
+
+The ownCloud Web allows developers to implement apps for viewing and editing specific file types. For instance, the built-in preview app serves as the default application for opening media files like images, videos, or audio.
```

Comment:
"The ownCloud Web"  sounds weird

Reply:
I agree, just copied it over from the other sections. Simply "ownCloud Web" would sound better though IMO, I'll adjust it.